docs/new_services.rst

```diff
@@ -0,0 +1,41 @@
+.. _topics-new_services:
+
+New Services
+============
+
+"Service" refers generally to any process, possibly long-running, which you
+want to run on the test cluster.
+
+These can be services you would actually deploy
+(e.g., Kafka brokers, ZK servers, REST proxy) or processes used during testing
+(e.g. producer/consumer performance processes). You should also make each
```

Comment:
Rather than "you should", saying something like "Services that are distributed systems can support a variable number of nodes, allowing one implementation to handle a variety of tests" might give Service authors a bit more flexibility. In at least a few cases, there really isn't any way to support more than one instance at a time.